modules/channel-web/src/backend/api.ts

```diff
@@ -318,7 +318,9 @@ export default async (bp: typeof sdk, db: Database) => {
       const conversation = await req.messaging.getConversationById(conversationId)
       const messages = await req.messaging.listMessages(conversationId, config.maxMessagesHistory)
 
-      return res.send({ ...conversation, messages })
+      const filteredMessages = messages.filter(m => m.payload.type !== 'visit')
```

Comment:
Instead of filtering here, what do you think of adding a new parameter when querying messaging (ignoredPayloadTypes)  ? This would return smaller payloads and would be quicker

Reply:
This can't work because messaging doesn't have a message type row to use as a filter